### PR TITLE
LPS-197153 Fix Picker focus after changing selections and improvements in the text for screen readers

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
Focus was lost when selecting an options in the page editor language selector. Also, the sr message on each option should be updated from Language en-US: Default to en-US Language: Default .
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-197153)

Steps to Verify:
----------------
1. Open the page editor
1. Select a language using the keyboard
1. Use a screen reader to check that the new text is read. 

Screenshots:
-----------------------
Captura de pantalla 2023-09-25 a las 16 27 47

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -9424,6 +9424,7 @@ language-ids=Language IDs
 language-key-information=Language Key Information
 language-labels=Language Labels
 language-labels-help=Represents the list of converted labels that are displayed through the application.
+language-option-label-and-status-x-x={0} Language: {1}
```
Comment (victorg1991) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:9427]: Hey @claraizquierdo I believe this has 0% changes to pass to master, can we compose this using two language keys? This is going to be rejected by brian

Comment (claraizquierdo) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:9427]: Hi @victorg1991 , this language key allow RTL languages like Arabic to properly change the order so it makes sense.

**Comment by victorg1991** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/141363

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**